misc/Makefile

```diff
@@ -15,17 +17,45 @@ doc/qutebrowser.1.html:
 
 install: doc/qutebrowser.1.html
 	$(PYTHON) setup.py install --prefix="$(PREFIX)" --optimize=1 $(SETUPTOOLSOPTS)
+	$(PYTHON) setup.py install --mandir="$(datarootdir)/man" --optimize=1 $(SETUPTOOLSOPTS)
```

Comment:
When I try to run this, I get:

```
...
Installing qutebrowser script to tmpinstall/usr/local/bin
python3 setup.py install --mandir="dataroot//man" --optimize=1 --root="tmpinstall/"
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --mandir not recognized
```

I don't think that --mandir/datadir is needed in setup.py, I think that all we need to use it for is copying the manfiles over.

By the way, I'm testing this with:

```
make -f misc/Makefile -C . install DESTDIR=tmpinstall/ <OTHER VARIABLES>
```

which I'm not sure is "correct" but seems good enough to me :P

Reply:
well, looks like --mandir (command) needs to be implemented, i might need some assistance on which exact file i should implement this?

Reply:
well thanks for the comments but how can i use "make -f misc/Makefile -C . install DESTDIR=tmpinstall/ <OTHER VARIABLES>" to test the code.

Reply:
@Hackingforlife, the file that that flag is passed to is [setup.py](https://github.com/qutebrowser/qutebrowser/blob/master/setup.py), however, I don't think that that's actually needed. setup.py is used for installing python dependencies, and I think it might be easier to just use the makefile commands to copy the man files into their proper places (as you are doing right below!).

@bamwireku, if you at at a terminal or shell and `cd` to the directory qutebrowser is in, you should be able to paste that command and have the install run. You'll need to have make installed though (through your package manager if you're on linux, or brew/macports if you're on mac os). Here's a little paste of me demonstrating it, let me know if you're stuck on this (this is from master):

```
[jay@laythe ~]$ cd ~/Code/qutebrowser                                                                                                                                                               [42471] 10:56AM
[jay@laythe qutebrowser]$ which make                                                                                                                                                      [master↓] [42472] 10:56AM
/usr/bin/make
[jay@laythe qutebrowser]$ # make is installed!                                                                                                                                            [master↓] [42473] 10:56AM
[jay@laythe qutebrowser]$ make -f misc/Makefile -C . install DESTDIR=tmpinstall/                                                                                                          [master↓] [42474] 10:56AM
make: Entering directory '/home/jay/Code/qutebrowser'
a2x -f manpage doc/qutebrowser.1.asciidoc
python3 setup.py install --prefix="/usr/local" --optimize=1 --root="tmpinstall/"
running install
running build
running build_py
.......
<lots of output>
.......
[jay@laythe qutebrowser]$ 
```
and after this, the `tempinstall` directory has an installation of qutebrowser in it.